client/src/modules/admin/components/admin-edit.component.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import Modal from 'react-bootstrap/Modal';
+
+import Label from '../../core/components/label.component';
+import Button from '../../core/components/button.component';
+import InputError from '../../core/components/input_error.component';
+import { validationSchema } from '../admin.schema';
+import { useDispatch, useSelector } from 'react-redux';
+import { getAdmin, updateAdmin } from '../admin.actions';
+import { toast } from 'react-toastify';
```

Comment:
This is a node module import! this imports will go to the top of the import list